src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -452,23 +465,38 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
 
         List<EmailWrapper> emails = new ArrayList<EmailWrapper>();
 
+        /* The instructor preamble aims at informing the instructor about an email that has been sent to the students
+        of his course */
+        String instructorPreamble = generateInstructorPreamble(course.getId(), course.getName());
+
+        //The corresponding preamble for students is blank since these are the recipients of the initial email.
+        String studentPreamble = "";
+
         for (InstructorAttributes instructor : instructors) {
-            emails.add(generateFeedbackSessionClosedEmail(course, session, instructor.name, instructor.email));
+            emails.add(generateFeedbackSessionClosedEmail(course, session, instructor.name, instructor.email,
+                    instructorPreamble));
         }
         for (StudentAttributes student : students) {
-            emails.add(generateFeedbackSessionClosedEmail(course, session, student.name, student.email));
+            emails.add(generateFeedbackSessionClosedEmail(course, session, student.name, student.email,
+                    studentPreamble));
         }
 
         return emails;
     }
 
+    /**
+     * Generates the feedback session closed email for both the instructors and the students with the given
+     * {@code userName} and {@code userEmail}.
+     */
     private EmailWrapper generateFeedbackSessionClosedEmail(CourseAttributes course,
-            FeedbackSessionAttributes session, String userName, String userEmail) {
+            FeedbackSessionAttributes session, String userName, String userEmail,
+            String instructorFragment) {
```

Comment:
I feel this should be `instructorPreamble` or just `preamble` but I'll leave @wkurniawan07 to decide based on consistency with other similar cases.